sdk/src/account/address.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the Aleo SDK library.
+
+// The Aleo SDK library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The Aleo SDK library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
```

Comment:
Well there is an issue with the license transparency because I was looking into js sdk/wasm implementation so this code must be gpl 2 but python-sdk currently licensed under Apache 2. What shall I do?

Reply:
I think we'll end up changing this to the GPL license for the python SDK. Thanks for clarifying.

Reply:
Well I am trying to find any mentions of Apache license and I can't find any. Maybe I was looking into another project :fearful:

Reply:
Yeah, the python-sdk has already GPL 3 license, so it should all be fine. Thanks for your great work, we're reviewing it!